governance/polls/Ratification Poll - Onboard Task Force - Growth Task Force (MIP75c3-SP1) - June 13, 2022.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ratification Poll for Onboard Growth Task Force (MIP75c3-SP1) - June 13, 2022
+summary:  The Growth Task Force aims to grow the Maker ecosystem as fast as possible.
+discussion_link: https://forum.makerdao.com/t/mip75c3-sp1-onboard-task-force-growth-task-force/15094
+vote_type: Plurality Voting
+categories:
+   - MIPs
+   - High Impact  
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-27T16:00:00
+---
+# Ratification Poll for Onboard Growth Task Force (MIP75c3-SP1) - June 13, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP75c3-SP1 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*It is our view that we are in a nascent stage of crypto, and controlling as much future market share as possible is critical for the survival of this project. Our focus is on practical solutions to achieve the objectives listed below as quickly as possible. Perfection is the enemy of progress. We aim to find solutions that work first and backfill decentralization / resilience later.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/24779f52630df945595594759e4cb4b757f4c177)
```

Comment:
Use this URL: https://github.com/makerdao/mips/blob/24779f52630df945595594759e4cb4b757f4c177/MIP75/MIP75c3-Subproposals/MIP75c3-SP1.md